includes/blocks/sophi-page-list-item/block.json

```diff
@@ -0,0 +1,47 @@
+{
+  "title": "Sophi Recommended Post",
+  "name": "sophi/page-list-item",
+  "attributes": {
+    "postUpdated": {
+      "type": "boolean"
+    },
+    "postTitle": {
+      "type": "string"
+    },
+    "postID": {
+      "type": "integer"
+    },
+    "postLink": {
+      "type": "string"
+    },
+    "postExcept": {
+      "type": "string"
+    },
+    "postAuthor": {
+      "type": "string"
+    },
+    "postDate": {
+      "type": "string"
+    },
+    "postDateC": {
+      "type": "string"
+    },
+    "featuredImage": {
+      "type": "string"
+    },
+    "linkToFeaturedImage": {
+      "type": "boolean"
+    },
+    "overrideRule": {
+      "type": "string"
+    },
+    "overridePostID": {
+      "type": "integer"
+    },
+    "overrideExpiry": {
+      "type": "integer"
+    }
+  },
```

Comment:
@faisal-alvi In order to hide the lock icon for the block you can add 

```json
{
    "supports": {
        "lock": false
    }
}
```

With that option, the lock still happens but the UI for it doesn't get shown. 

https://gutenberg.10up.com/reference/Blocks/block-supports/#lock